src/apps/loan-list/materials/utils/digital-material-fetch-hoc.tsx

```diff
@@ -35,7 +35,13 @@ const fetchDigitalMaterial =
         data: productsData,
         isSuccess: isSuccessDigital,
         isLoading
-      } = useGetV1ProductsIdentifier(item.identifier);
+      } = useGetV1ProductsIdentifier(item.identifier, {
+        query: {
+          // We never want to pass an empty string to the API
+          // So we only enable the query if we have an isbn
+          enabled: !!item.identifier
+        }
+      });
```

Comment:
This check may be unnecessary given the check on L30 but I guess it is nice to have around given the problems caused if we allow an empty string to pass.